exonum/src/api/private/system.rs

```diff
@@ -186,8 +195,40 @@ impl Api for SystemApi {
             self_.ok_response(&::serde_json::to_value(info).unwrap())
         };
 
+        let self_ = self.clone();
+        let consensus_enabled_info = move |_: &mut Request| -> IronResult<Response> {
+            let info = self_.get_consensus_enabled_info();
+            self_.ok_response(&::serde_json::to_value(info).unwrap())
+        };
+
+        let self_ = self.clone();
+        let consensus_enabled_set = move |req: &mut Request| -> IronResult<Response> {
+            let map = req.get_ref::<Params>().unwrap();
+            match map.find(&["enabled"]) {
+                Some(&ParamsValue::Boolean(enabled)) => {
+                    self_.set_consensus_enabled_info(enabled)?;
+                    self_.ok_response(&::serde_json::to_value("Ok").unwrap())
+                }
+                _ => {
+                    Err(ApiError::IncorrectRequest(
+                        "Required boolean parameter 'enabled' is missing".into(),
+                    ))?
+                }
+            }
+        };
+
         router.get("/v1/peers", peers_info, "peers_info");
         router.post("/v1/peers", peer_add, "peer_add");
         router.get("/v1/network", network, "network_info");
+        router.get(
```

Comment:
This is subjective, but I don't quite like single-purpose minimalistic endpoints if there is no way to aggregate them (e.g., GraphQL). In this particular case, it seems that the consensus switch could be included into the `/v1/network` endpoint.

Reply:
Yeah, but this is supposed to be a just temporary solution for node's control.